### ISSUE TITLE
Unexpected result of Training from scratch

### ISSUE THREAD

**Issue by wenwen12321** [paTRICK-swk/P-STMO#11]
Hi author, 
Thanks for your excellent work!

I can evaluate your model with your pre-trained model.
But when I tried to follow your train from scratch guideline (For the pre-training stage on Human3.6M)

`
python run.py -f 243 -b 160 --MAE --train 1 --layers 3 -tds 2 -tmr 0.8 -smn 2 --lr 0.0001 -lrd 0.97
`

the train.log is like 

> 2022/08/12 17:44:17 epoch: 1, lr: 0.0001000, loss: 0.0814, loss_test: 0.0305, p1: 30.54, p2: 0.00
2022/08/12 18:55:30 epoch: 2, lr: 0.0000970, loss: 0.0367, loss_test: 0.0245, p1: 24.47, p2: 0.00
2022/08/12 20:06:40 epoch: 3, lr: 0.0000941, loss: 0.0324, loss_test: 0.0212, p1: 21.22, p2: 0.00
2022/08/12 21:17:54 epoch: 4, lr: 0.0000913, loss: 0.0303, loss_test: 0.0193, p1: 19.25, p2: 0.00
2022/08/12 22:29:07 epoch: 5, lr: 0.0000885, loss: 0.0292, loss_test: 0.0184, p1: 18.41, p2: 0.00
2022/08/12 23:40:19 epoch: 6, lr: 0.0000859, loss: 0.0282, loss_test: 0.0183, p1: 18.39, p2: 0.00
2022/08/13 00:51:31 epoch: 7, lr: 0.0000833, loss: 0.0274, loss_test: 0.0176, p1: 17.70, p2: 0.00
2022/08/13 02:02:43 epoch: 8, lr: 0.0000808, loss: 0.0268, loss_test: 0.0170, p1: 17.09, p2: 0.00
2022/08/13 03:13:56 epoch: 9, lr: 0.0000784, loss: 0.0262, loss_test: 0.0169, p1: 17.00, p2: 0.00
2022/08/13 04:25:09 epoch: 10, lr: 0.0000760, loss: 0.0258, loss_test: 0.0169, p1: 17.04, p2: 0.00
2022/08/13 05:36:21 epoch: 11, lr: 0.0000737, loss: 0.0255, loss_test: 0.0162, p1: 16.26, p2: 0.00
2022/08/13 06:47:33 epoch: 12, lr: 0.0000715, loss: 0.0252, loss_test: 0.0168, p1: 16.98, p2: 0.00
2022/08/13 07:58:53 epoch: 13, lr: 0.0000694, loss: 0.0249, loss_test: 0.0159, p1: 16.04, p2: 0.00
2022/08/13 09:11:11 epoch: 14, lr: 0.0000673, loss: 0.0247, loss_test: 0.0160, p1: 16.20, p2: 0.00
2022/08/13 10:22:49 epoch: 15, lr: 0.0000653, loss: 0.0244, loss_test: 0.0149, p1: 15.07, p2: 0.00
2022/08/13 11:34:02 epoch: 16, lr: 0.0000633, loss: 0.0243, loss_test: 0.0145, p1: 14.70, p2: 0.00
...

I am confused that why p2 is always 0? and why p1 is lower than expected.
Could you please release your Training from scratch (pre-training & fine-tuning) on Human3.6M's train.log for me?

Thanks a lot.

**Comment by paTRICK-swk** [comment]
p2 stands for protocol#2, which is only evaluated in Stage 2. The training log of Stage 1 is provided below.

> 2022/01/11 04:18:19 epoch: 1, lr: 0.0001000, loss: 0.0690, loss_test: 0.0281, p1: 28.11, p2: 0.00
2022/01/11 04:55:41 epoch: 2, lr: 0.0000950, loss: 0.0347, loss_test: 0.0239, p1: 23.90, p2: 0.00
2022/01/11 05:33:12 epoch: 3, lr: 0.0000902, loss: 0.0317, loss_test: 0.0208, p1: 20.71, p2: 0.00
2022/01/11 06:10:40 epoch: 4, lr: 0.0000857, loss: 0.0299, loss_test: 0.0187, p1: 18.70, p2: 0.00
2022/01/11 06:48:14 epoch: 5, lr: 0.0000815, loss: 0.0288, loss_test: 0.0179, p1: 17.97, p2: 0.00
2022/01/11 07:25:57 epoch: 6, lr: 0.0000774, loss: 0.0278, loss_test: 0.0174, p1: 17.47, p2: 0.00
2022/01/11 08:03:28 epoch: 7, lr: 0.0000735, loss: 0.0271, loss_test: 0.0175, p1: 17.58, p2: 0.00
2022/01/11 08:40:52 epoch: 8, lr: 0.0000698, loss: 0.0265, loss_test: 0.0165, p1: 16.58, p2: 0.00
2022/01/11 09:18:23 epoch: 9, lr: 0.0000663, loss: 0.0260, loss_test: 0.0166, p1: 16.72, p2: 0.00
2022/01/11 09:55:46 epoch: 10, lr: 0.0000630, loss: 0.0256, loss_test: 0.0167, p1: 16.81, p2: 0.00
2022/01/11 10:32:43 epoch: 11, lr: 0.0000599, loss: 0.0253, loss_test: 0.0161, p1: 16.19, p2: 0.00
2022/01/11 11:09:42 epoch: 12, lr: 0.0000569, loss: 0.0250, loss_test: 0.0168, p1: 16.95, p2: 0.00
2022/01/11 11:46:39 epoch: 13, lr: 0.0000540, loss: 0.0247, loss_test: 0.0160, p1: 16.12, p2: 0.00
2022/01/11 12:23:41 epoch: 14, lr: 0.0000513, loss: 0.0246, loss_test: 0.0156, p1: 15.77, p2: 0.00
2022/01/11 13:00:40 epoch: 15, lr: 0.0000488, loss: 0.0244, loss_test: 0.0161, p1: 16.23, p2: 0.00
2022/01/11 13:37:38 epoch: 16, lr: 0.0000463, loss: 0.0242, loss_test: 0.0149, p1: 15.09, p2: 0.00
2022/01/11 14:14:37 epoch: 17, lr: 0.0000440, loss: 0.0241, loss_test: 0.0162, p1: 16.29, p2: 0.00
2022/01/11 14:51:44 epoch: 18, lr: 0.0000418, loss: 0.0239, loss_test: 0.0164, p1: 16.61, p2: 0.00
2022/01/11 15:29:22 epoch: 19, lr: 0.0000397, loss: 0.0238, loss_test: 0.0170, p1: 17.18, p2: 0.00
2022/01/11 16:07:03 epoch: 20, lr: 0.0000377, loss: 0.0236, loss_test: 0.0163, p1: 16.42, p2: 0.00
2022/01/11 16:44:28 epoch: 21, lr: 0.0000358, loss: 0.0236, loss_test: 0.0157, p1: 15.86, p2: 0.00
2022/01/11 17:21:58 epoch: 22, lr: 0.0000341, loss: 0.0234, loss_test: 0.0178, p1: 17.94, p2: 0.00
2022/01/11 17:59:09 epoch: 23, lr: 0.0000324, loss: 0.0233, loss_test: 0.0168, p1: 16.91, p2: 0.00
2022/01/11 18:35:40 epoch: 24, lr: 0.0000307, loss: 0.0233, loss_test: 0.0162, p1: 16.32, p2: 0.00
2022/01/11 19:12:14 epoch: 25, lr: 0.0000292, loss: 0.0231, loss_test: 0.0172, p1: 17.38, p2: 0.00
2022/01/11 19:48:53 epoch: 26, lr: 0.0000277, loss: 0.0231, loss_test: 0.0160, p1: 16.11, p2: 0.00
2022/01/11 20:25:33 epoch: 27, lr: 0.0000264, loss: 0.0231, loss_test: 0.0179, p1: 18.12, p2: 0.00
2022/01/11 21:02:16 epoch: 28, lr: 0.0000250, loss: 0.0230, loss_test: 0.0175, p1: 17.63, p2: 0.00
2022/01/11 21:38:39 epoch: 29, lr: 0.0000238, loss: 0.0230, loss_test: 0.0186, p1: 18.80, p2: 0.00
2022/01/11 22:15:10 epoch: 30, lr: 0.0000226, loss: 0.0228, loss_test: 0.0171, p1: 17.32, p2: 0.00
2022/01/11 22:51:42 epoch: 31, lr: 0.0000215, loss: 0.0228, loss_test: 0.0173, p1: 17.46, p2: 0.00
2022/01/11 23:27:59 epoch: 32, lr: 0.0000204, loss: 0.0228, loss_test: 0.0175, p1: 17.70, p2: 0.00
2022/01/12 00:04:17 epoch: 33, lr: 0.0000194, loss: 0.0227, loss_test: 0.0179, p1: 18.12, p2: 0.00
2022/01/12 00:40:37 epoch: 34, lr: 0.0000184, loss: 0.0227, loss_test: 0.0178, p1: 18.01, p2: 0.00
2022/01/12 01:17:19 epoch: 35, lr: 0.0000175, loss: 0.0226, loss_test: 0.0181, p1: 18.27, p2: 0.00
2022/01/12 01:53:26 epoch: 36, lr: 0.0000166, loss: 0.0226, loss_test: 0.0179, p1: 18.05, p2: 0.00
2022/01/12 02:29:41 epoch: 37, lr: 0.0000158, loss: 0.0225, loss_test: 0.0189, p1: 19.13, p2: 0.00

And the log of Stage 2 can be found from [here](https://github.com/paTRICK-swk/P-STMO/issues/8#issuecomment-1214581045).

**Comment by wenwen12321** [comment]
@paTRICK-swk, Thanks for your prompt reply.
I see! I think I misunderstood your training process.
